test/colony.js

```diff
@@ -95,7 +103,7 @@ contract("Colony", addresses => {
       assert.equal(colonyBalance.toNumber(), 1);
     });
 
-    it("should take colony network as an owner", async () => {
+    it("should set colony network to be owner", async () => {
```

Comment:
Like I mentioned in the contract logic above, I don't understand this decision. After the colony is created, its permissions should be contained to the creator and the network should no longer have any control over it.